packages/dev/core/src/Misc/fileTools.ts

```diff
@@ -216,9 +220,22 @@ export const LoadImage = (
     const img = new Image();
     SetCorsBehavior(url, img);
 
+    const handlersList: { target: any; name: string; handler: any }[] = [];
+
+    const loadHandlersList = () => {
+        handlersList.forEach((handler) => {
+            handler.target.addEventListener(handler.name, handler.handler);
+        });
+    };
+
+    const unloadHandlersList = () => {
+        handlersList.forEach((handler) => {

```

Comment:
Just to stay safe, after removing the event listeners, you should reset the array. Or maybe instead of forEach you can pop each handler and remove it.